tests-bundle/1.7/test_1dot7.py

```diff
@@ -102,3 +102,7 @@ async def test_deploy(ops_test: OpsTest, lightkube_client, deploy_cmd):
         timeout=from_minutes(minutes=30),
         idle_period=from_minutes(minutes=3),
     )
+
+    await ops_test.model.applications["kubeflow-profiles"].units[0].run_action(
+        "create-profile", profilename="demo-namespace", username=USERNAME
+    )
```

Comment:
minor suggestion, just since `Profile` and `Namespace` are different here but easy to conflate

```suggestion
    await ops_test.model.applications["kubeflow-profiles"].units[0].run_action(
        "create-profile", profilename=USERNAME, username=USERNAME
    )
```

The profilename could also be `demo-profile` or something else, too.  Just felt like "namespace" in there was confusing

Reply:
I think we need something to distinguish between username and namespaces that will be created.
I am open for ideas but do not want to have the same profile name as username 

![image](https://github.com/canonical/bundle-kubeflow/assets/51964909/ef04a906-0d6b-410a-a4f2-e77d8dc66eae)

Reply:
in big org installs, it is common to have username be the user's email (since many password systems use email address as username), then the user's profile be everything left of the "@" as a default.  So:
* `profile = user123@email.com`
* `username = user123`

How about that here?

Reply:
love it!